Jenkinsfile

```diff
@@ -25,7 +25,7 @@ pipeline {
         )
         booleanParam(
             name: 'REFRESH_DB',
-            defaultValue: false,
+            defaultValue: true,
```

Comment:
In my limited understanding, I assumed we needed to enable a database refresh in order for the application to have any pages so I toggled this flag so I could make assumptions about page layout and such. Do the docker deploys come with a base set of pages already configured in the app?

Reply:
I am pretty sure that the first deployment of a stack will grab the latest DB dump, regardless of this setting, and that the setting only affects subsequent deployments. But we should confirm that.

Reply:
Yeah, here's where the logic is applied:

https://github.com/cfpb/cfgov-refresh/blob/1e6b10c80fa01d18b627f5ef501cdb293f5ba2aa/docker-stack.yml#L76-L77

So if the database is empty OR if the `REFRESH_DB` param is true, it will run the refresh data script.

Reply:
More than a lot of time, this setting prevents an automatic refresh of the database from wiping out any content relevant to the PR/branch has may have been created (unless it's run with this manually checked).

Reply:
Reverted this change, thanks for the clarification